doc/modules/mixture.rst

```diff
@@ -211,49 +232,67 @@ Cons
 
 .. _dpgmm:
 
-DPGMM: Infinite Gaussian mixtures
-=================================
+Dirichlet Process: Infinite Gaussian mixture
+============================================
```

Comment:
Please make this a sub-section of "Variational Bayesian Gaussian Mixture" by using "-------" underline.